ble/services/EddystoneService.h

```diff
@@ -185,6 +185,29 @@ class EddystoneService
         return false;
     }
 
+    /**
+     *  Set Eddystone URL Frame information.
+     *  @param[in] power          TX Power in dB measured at 0 meters from the device.
+     *  @param[in] url            Encoded URL
+     *  @param[in] urlAdvPeriodIn How long to advertise the URL frame (measured in # of adv periods)
+     *  @return false on success, true on failure.
```

Comment:
when returning bools, it is more customary to return true on success and false on failure.